go.mod

```diff
@@ -28,3 +28,5 @@ require (
 	gopkg.in/yaml.v2 v2.4.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/flyteorg/flyteidl => github.com/flyteorg/flyteidl v0.18.51-0.20210529142634-db6e9d4c701a
```

Comment:
we should first check in flyteild

Reply:
Sure . Once this is approved and checkedin https://github.com/flyteorg/flyteidl/pull/176 i will update this file